### DISCUSSION TITLE
how do I bypass my school email filters???

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1332] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
e???

**Comment by v4pl** [comment, marked as answer]
Um u cant unless u use your Gmail in like skiovox or unenrollment.

**Reply by ghost** [reply]
True

**Comment by Luigi-Master** [comment]
Just make an Outlook account on your Chromebook.

**Reply by ghost** [reply]
but I need to use my school email for github beniefts

**Reply by Luigi-Master** [reply]
What benefits? Its the same thing as having an Outlook account, Its just owned by Microsoft?

**Comment by schoolexploitkid** [comment]
make outlook account, get a pc, use that to make a new google account
